src/test/java/org/isf/generaldata/GeneralDataTest.java

```diff
@@ -0,0 +1,20 @@
+package org.isf.generaldata;
```

Comment:
Please add the standard license header to all new files; thanks.   The other alternative is to run the license plugin periodically to ensure all headers are present.

Reply:
Done, thanks!!!